src/components/CardSuccess.tsx

```diff
@@ -1,13 +1,10 @@
-import { Dispatch, SetStateAction } from "react";
-import { FormActions } from "../reducers/form-reducer";
 import { EFormaActions } from "../enums/Form";
+import { useForm } from "../hooks/useForm";
 
-type FormProps = {
-  dispatch: Dispatch<FormActions>;
-  setIsSubmitted: Dispatch<SetStateAction<boolean>>;
-};
+const CardSuccess = () => {
+
+  const { dispatch, setIsSubmitted } = useForm()
 
-const CardSuccess = ({ dispatch, setIsSubmitted }: FormProps) => {
   const handleClick = () => {
```

Comment:
this handleChange should be ideal to have it in the context so you remove most of the logic to the context and keep clean the component